client/lib/abtest/index.js

```diff
@@ -87,6 +88,16 @@ ABTest.prototype.init = function( name ) {
 		throw new Error( 'A default variation is specified for ' + name + ' but it is not part of the variations' );
 	}
 
+	const languageSlugs = map( config( 'languages' ), 'langSlug' );
+
+	if (
+		testConfig.localeTargets &&
+		! every( testConfig.localeTargets, ( target ) => {
+			return languageSlugs.indexOf( target ) !== -1;
+		} ) ) {
```

Comment:
Bear with as I muse/ramble. So, this throws an error if any `localTargets` are invalid (not in `languageSlugs`), correct? My first thought is that this `if` is certainly-not-impossible-but-a-little-cryptic to passerbys.  What about wrapping the test conditions in a simple function for readablity?  eg: 

`if ( this.testHasInvalidLocales( testConfig.localeTargets ) )`